### DISCUSSION TITLE
Simplify `dataExtraction`->`extract` and `applyTransform`->`transform` in the JSON-LD Croissant syntax

### DISCUSSION THREAD

**Discussion by marcenacp** [mlcommons/croissant#151] [Ideas]
`data` in `dataExtraction` and `apply` in `applyTransform` are broad terms that do not add to the understanding of what they do.

I propose to simplify them:
- `dataExtraction`->`extract`
- `applyTransform`->`transform`

This also echoes to ETL (Extract Transform Load) which are known notions/terms for data engineers. WDYT?

cc @ccl-core @benjelloun @pierrot0

**Comment by benjelloun** [comment]
Your proposal makes sense to me.

Tangentially, I don't really like the current `distribution` field. I thought we were going to use `file` and `fileset` instead of `distribution`... Is that just an intermediary step?

**Reply by marcenacp** [reply]
This is not an intermediary step. During the implementation, it didn't make sense to distinguish between fileobject and fileset. What does this distinction bring? We could do a new migration.

**Comment by pierrot0** [comment]
Proposal LGTM.

**Comment by marcenacp** [comment]
Done in https://github.com/mlcommons/croissant/pull/196.
